app/models/effective/tag.rb

```diff
@@ -0,0 +1,26 @@
+module Effective
+  class Tag < ApplicationRecord
+    self.table_name = (EffectivePages.tags_table_name || :tags).to_s
+
+    log_changes if respond_to?(:log_changes)
+
+    has_many :taggings, class_name: 'Effective::Tagging', dependent: :delete_all
```

Comment:
does this need accepts_nested_attributes?

Reply:
I've added in the `acts_as_tagged` concern, do you think we need in the Tag model?